config/locales/curation_concerns.en.yml

```diff
@@ -101,17 +102,29 @@ en:
         property: keywords
       title:
         property: name
+
```

Comment:
It's from https://github.com/projecthydra-labs/curation_concerns/pull/716/files/212f7496bcf388262e5ff46219722043cd319f21#diff-a47c94c05ae8b7b630d2d8c1ee12ff3d right?